recipe/meta.yaml

```diff
@@ -12,14 +12,11 @@ source:
     - no-console.patch
 
 build:
-  number: 2
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py36]
+  number: 4
 
 requirements:
   build:
+    - {{ compiler('cxx') }}
     - toolchain
```

Comment:
Don't need this explicitly anymore if you're using `{{ compiler() }}`; it's pulled in automatically when appropriate and not used when not needed.